articles/closing_the_trust_gap/part1.md

```diff
@@ -73,6 +73,6 @@ show such a confirmation dialog and avoid things like collisions in
 the four byte directory. It can even be decentralized!
 
 Read about what needs to be done (it's not much!) and by who
-(mostly all of us, unfortunately) in the [next part](./part2) of this series.
```

Comment:
This is meant to be read at https://chriseth.github.io/notes/articles/closing_the_trust_gap/part1 - so I think the correct way is without the `.md` suffix.

Reply:
Oh I see, sorry I was reading it on github.

Reply:
Should I close the PR then?

Reply:
Could you change it so that it only includes the other link fixes? That would be great!

Reply:
Of course.